src/main/js/component/general/audit/Index.js

```diff
@@ -76,9 +77,14 @@ class Index extends Component {
         }).then((response) => {
             this.setState({ inProgress: false });
             if (!response.ok) {
-                return response.json().then((json) => {
-                    this.setState({ message: json.message });
-                });
+                switch(response.status) {
+                    case 401:
```

Comment:
I could be wrong, but it looks like for 401 we do nothing, and for 403 we logout. Is that what is happening here?

Reply:
case 401:
case 403: 
together both 401 and 403 are treated the same.  That's how you group cases in switch statements.  We want the same behavior for both 401 and 403